tools/POTools/Services/Resources/ResXFileReader.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace POTools.Services.Resources;
 
-public class ResXFileReader : IEnumerable<KeyValuePair<string, string>>
+public class ResXFileReader : IEnumerable<(string name, string value, string comment)>
```

Comment:
nit: formatting (in most cases I use pascal case for tuple member names)

```suggestion
public class ResXFileReader : IEnumerable<(string Name, string Value, string Comment)>
```

Reply:
Ok. Seems opinions vary here, but sure!